server/odc-service/src/main/java/com/oceanbase/odc/service/task/caller/PodConfig.java

```diff
@@ -59,4 +59,8 @@ public class PodConfig {
     private Long mountDiskSize;
 
     private Long maxNodeCount;
+
+    private Long nodeCpu;
+
+    private Long nodeMem;
```

Comment:
if hardcode unit, filed name may end with unit name.

better follow k8s configuration design.

Reply:
okey, i rename to `nodeMemInMb` which follow other field